tensorflow/g3doc/get_started/basic_usage.md

```diff
@@ -292,4 +292,3 @@ it. See the
 [MNIST fully-connected feed tutorial](../tutorials/mnist/tf/index.md)
 ([source code](https://www.tensorflow.org/code/tensorflow/g3doc/tutorials/mnist/fully_connected_feed.py))
 for a larger-scale example of feeds.
-
```

Comment:
This one is a bit questionable. Many tools like vim add new line at the end. For that reason I believe it is better to keep it there. One blank at the end.